packages/presentation/lib/screen/details/details_bloc.dart

```diff
@@ -10,8 +11,12 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 abstract class DetailsBloc extends Bloc<DetailsScreenArguments, DetailsData> {
   factory DetailsBloc(
     GetCastUseCase getCastUseCase,
+    LogAnalyticsEventUseCase logAnalyticsEventUseCase,
```

Comment:
move to base bloc
no need to inject it in each screen as u'll use it everywhere

Reply:
Not really everywhere. For example there is no action on Splash screen. Should it be in the base bloc anyway?

Reply:
I think SplashScreen no worse than any other. Don't u log any event here?
In practice analytics exists everywhere. Even if 9/10 screens(except splash) use analytics, then we move it in base bloc